spyder/config/main.py

```diff
@@ -453,8 +453,10 @@
               # ---- In widgets/arraybuider.py ----
               'array_builder/enter array inline': "Ctrl+Alt+M",
               'array_builder/enter array table': "Ctrl+M",
-              # ---- In widgets/variableexplorer/aarayeditor.py ----
+              # ---- In widgets/variableexplorer/arrayeditor.py ----
               'variable_explorer/copy': 'Ctrl+C',
+              # ---- In widgets/variableexplorer/namespacebrowser.py ----
+              'variable_explorer/search': 'Ctrl+Alt+F',

```

Comment:
Why can't we use `Ctrl+F` here?

Reply:
It collides with the shortcut of the find widget (a message appears when opening the shortcut preferences with `Ctrl+F`):

![imagen](https://user-images.githubusercontent.com/16781833/61817111-64302800-ae13-11e9-9a51-471c620c24ec.png)

Reply:
Then you need to apply the shortcut at the widget level and not at the plugin level (I think). For that please take a look at `register_widget_shortcuts` in `api/plugins.py`

Reply:
At the end the problem was that the find replace shortcut was in the global context (`_`)